www-client/elinks/elinks-0.13_p20180225.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads"
+inherit autotools python-any-r1 vcs-snapshot
+
+COMMIT="0dd3aaabe61fbc5ed0d9a74915678d11d00c2a1c"
+
+DESCRIPTION="Advanced and well-established text-mode web browser"
+HOMEPAGE="http://elinks.or.cz/"
+SRC_URI="https://github.com/rkd77/felinks/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
+IUSE="bittorrent brotli bzip2 debug finger ftp gopher gpm guile idn ipv6
+	  javascript libressl lua +mouse nls nntp perl ruby samba ssl tre unicode X xml zlib"
+
+DEPEND="
+	${PYTHON_DEPS}
+	brotli? ( app-arch/brotli )
+	bzip2? ( >=app-arch/bzip2-1.0.2 )
+	gpm? ( >=sys-libs/ncurses-5.2:0= >=sys-libs/gpm-1.20.0-r5 )
+	guile? ( >=dev-scheme/guile-1.6.4-r1[deprecated] )
+	idn? ( net-dns/libidn )
+	javascript? ( >=dev-lang/spidermonkey-1.8.5:0= )
+	lua? ( >=dev-lang/lua-5:0= )
+	perl? ( dev-lang/perl:= )
+	ruby? ( dev-lang/ruby:* dev-ruby/rubygems:* )
+	samba? ( net-fs/samba )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	tre? ( dev-libs/tre )
+	X? ( x11-libs/libX11 x11-libs/libXt )
+	xml? ( >=dev-libs/expat-1.95.4 )
+	zlib? ( >=sys-libs/zlib-1.1.4 )"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-9999-parallel-make.patch
+	)
+
+src_prepare() {
+	default
+
+	sed -i -e 's/-Werror//' configure* || die
+
+	eautoreconf
+}
+
+src_configure() {
+	local myconf=""
```

Comment:
Please convert `myconf` into an array, and include all options here instead of doing the backslash hell. Both as part of live ebuild update.